### DISCUSSION TITLE
Welcome to discord-player-bot Discussions!

### DISCUSSION THREAD

**Discussion by aka-pal** [LakhindarPal/discord-player-bot#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by theabsoluteray** [comment]
Does it work as a selfbot ?

**Reply by aka-pal** [reply]
If you mean **user account as bot**,  then no. It's against discord terms and conditions
